shelf_x: 1.25
shelf_y: 2.25
shelf_z: 0.75

shelf_box_x: 1.0
shelf_box_y: 1.0
shelf_box_z: 1.5